mathics/core/parser/operators.py

```diff
@@ -17,6 +17,8 @@
     'Exists': 240,
     'NotExists': 240,
     'Not': 230,
+    'Information': 5001,
+    'Definition' : 5000,
```

Comment:
Still not sure about this. What's the best? To assign the default precedence or to leave it as it is now? In principle, the last option seems to work.